distros/kubernetes/nvsentinel/charts/health-events-analyzer/values.yaml

```diff
@@ -41,25 +41,21 @@ logLevel: info
 
 config: |
   [[rules]]
-  name = "Rule1"
-  description = "Detect occurrence of XID13 -> XID31 within 30 minutes"
-  time_window = "30m"
-  recommended_action = "COMPONENT_RESET"
-  
-  [[rules.sequence]]
-  criteria = { "healthevent.entitiesimpacted.0.entitytype" = "GPU", "healthevent.entitiesimpacted.0.entityvalue" = "this.healthevent.entitiesimpacted.0.entityvalue", "healthevent.errorcode.0" = "13", "healthevent.nodename" = "this.healthevent.nodename" }
-  errorCount = 1
-  
+  name = "MultipleFatalError"
+  description = "Detect occurrence of multiple faults within 7 days"
+  time_window = "168h"
+  recommended_action = "REPORT_ISSUE"
+
   [[rules.sequence]]
-  criteria = { "healthevent.entitiesimpacted.0.entitytype" = "GPU", "healthevent.entitiesimpacted.0.entityvalue" = "this.healthevent.entitiesimpacted.0.entityvalue", "healthevent.errorcode.0" = "31", "healthevent.nodename" = "this.healthevent.nodename" }
-  errorCount = 1
+  criteria = {"healtheventstatus.faultremediated" = true, "healthevent.nodename" = "this.healthevent.nodename"}
+  errorCount = 5
 
   [[rules]]
-  name = "Rule2"
-  description = "Detect occurrence of XID13 3 times within 30 minutes"
-  time_window = "30m"
-  recommended_action = "CONTACT_SUPPORT"
+  name = "RepeatedXidError"
+  description = "Detect occurrence of fatal XIDs 5 times within 24 hours"
+  time_window = "24h"
+  recommended_action = "REPORT_ISSUE"
 
   [[rules.sequence]]
-  criteria = { "healthevent.entitiesimpacted.0.entitytype" = "GPU", "healthevent.entitiesimpacted.0.entityvalue" = "this.healthevent.entitiesimpacted.0.entityvalue", "healthevent.errorcode.0" = "13", "healthevent.nodename" = "this.healthevent.nodename" }
-  errorCount = 3 
+  criteria = { "healthevent.entitiesimpacted.0.entitytype" = "GPU","healthevent.entitiesimpacted.0.entityvalue" = "this.healthevent.entitiesimpacted.0.entityvalue","healthevent.errorcode.0" = "this.healthevent.errorcode.0","healthevent.nodename" = "this.healthevent.nodename","healthevent.checkname" = "{\"$ne\": \"HealthEventsAnalyzer\"}"}
```

Comment:
Isn't `"healthevent.checkname" = "{\"$ne\": \"HealthEventsAnalyzer\"}"` needed for every rule, can we add it as a default in the code rather than having the rule specify?